flink-connectors/flink-connector-kinesis/pom.xml

```diff
@@ -56,6 +58,18 @@ under the License.
 		</dependency>
 
 		<dependency>
+		    <groupId>org.apache.httpcomponents</groupId>
+		    <artifactId>httpclient</artifactId>
+		    <version>${apache.httpclient.version}</version>
+		</dependency>
+
+		<dependency>
+		    <groupId>org.apache.httpcomponents</groupId>
+		    <artifactId>httpcore</artifactId>
+		    <version>${apache.httpcore.version}</version>
+		</dependency>
```

Comment:
As Stephan mentioned, these shouldn't be needed to be declared. They'll already be pulled in due to the `promoteTransitiveDependencies` property in the shade config.

Adding them for the sake of hadoop-aws dependency doesn't make sense, for the same reason as before; we shouldn't assume that the Kinesis connector user will also use AWS S3.